slides/src/content/component.md

```diff
@@ -226,13 +229,13 @@ export class Child {
 
 ## Quiz (continued)
 
-1. Parent:  
```

Comment:
1. This change seems to have been brought in by accident (?).
2. The end-of-line double spaces are to force a line break - please use `<br>` instead.